src/map/skill.c

```diff
@@ -10645,11 +10651,22 @@ int skill_castend_pos2(struct block_list* src, int x, int y, uint16 skill_id, ui
 			break;
 
 		case WZ_METEOR:
+		case SU_CN_METEOR:
 			{
 				int area = skill->get_splash(skill_id, skill_lv);
 				short tmpx = 0, tmpy = 0, x1 = 0, y1 = 0;
 				int i;
 
+				/*
```

Comment:
Can you comment this out with `#if 0` instead? (and possibly add a comment mentioning why it's disabled)

I'm asking the `#if 0` because it doesn't cause the nested comment problems that `/*` would